proxygen_cli/cli/command_settings.py

```diff
@@ -15,14 +17,11 @@ def settings():
     """
     Get/set settings for the proxygen-cli.
     """
-    pass
+
 
 
 def _get_setting(key):
-    try:
-        return getattr(SETTINGS, key)
-    except AttributeError as e:
-        raise ValueError(f"No such setting: {key}")
+    return getattr(SETTINGS, key)
```

Comment:
Pydantic raises this error on initialisation of settings - this code is impossible to access